languages/python/jupyter/create_env.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+module load conda
+conda create -y -c conda-forge -n jupyterlab python=3.8 pandas scipy numpy matplotlib seaborn jupyterlab  
```

Comment:
suggestion to change env name to match sbatch_jupyter.sh, for consistency: 
```suggestion
conda create -y -c conda-forge -n jupyterenv python=3.8 pandas scipy numpy matplotlib seaborn jupyterlab  
```